startdash.py

```diff
@@ -1,4 +1,4 @@
-#!/opt/conda/envs/rapids/bin/python3
+#!/usr/bin/env python3
```

Comment:
Might need to edit the other python files this way as well? Can also do in a subsequent PR.

Reply:
I think, this might be the only entry-point script.